### PR TITLE
Remove GFE Overview table

### PR DESCRIPTION
**Pull request by juliusfitzhugh-ccpo**
### Delete `GFE Overview` table
- Delete `GFE Overview` column from `Acq. Package`

### Update components using GFE Overview
- `DOCGEN: Gather DescriptionOfWork Data`
  - remove references to `gfe_overview`

Ticket: AT-8227

### CODE REVIEW

File: f600233d1b154d507b782f84604bcb12/author_elective_update/sys_dictionary_x_g_dis_atat_acquisition_package_gfe_overview.xml [hunk 1]
```diff
@@ -0,0 +1,196 @@
+<?xml version="1.0" encoding="UTF-8"?><record_update>
+    <sys_dictionary action="DELETE" element="gfe_overview" table="x_g_dis_atat_acquisition_package">
```
Comment (juliusfitzhugh-ccpo) [inline f600233d1b154d507b782f84604bcb12/author_elective_update/sys_dictionary_x_g_dis_atat_acquisition_package_gfe_overview.xml:2]: Deleting the `GFE Overview` table first, appears to also remove the `gfe_overview` column from the `Acq. Package`.

File: f600233d1b154d507b782f84604bcb12/dictionary/x_g_dis_atat_acquisition_package.xml [hunk 2]
```diff
@@ -17,7 +17,6 @@
         <element label="Funding Plans" max_length="4000" name="funding_plans" reference="x_g_dis_atat_funding_plan" type="glide_list"/>
         <element label="Funding Request" max_length="32" name="funding_request" reference="x_g_dis_atat_funding_request" type="reference"/>
         <element label="Funding Requirement" max_length="32" name="funding_requirement" reference="x_g_dis_atat_funding_requirement" type="reference"/>
-        <element label="GFE Overview" max_length="32" name="gfe_overview" reference="x_g_dis_atat_gfe_overview" type="reference"/>
         <element label="IDIQ CLINs" max_length="4000" name="idiq_clins" reference="x_g_dis_atat_idiq_clin" type="glide_list"/>
```
Comment (juliusfitzhugh-ccpo) [inline f600233d1b154d507b782f84604bcb12/dictionary/x_g_dis_atat_acquisition_package.xml:20]: These changes appears to confirm the deletion of the `gfe_overview` column in the package. This was unexpected but appears to be a default behavior of SNOW. It was only noticed after looking for the `gfe_overview` column to delete in the `Acq. Package`.

File: diagrams.md [hunk 3]
```diff
@@ -361,14 +360,6 @@ erDiagram
         String description
         Integer order
     }
-    GFE-OVERVIEW {
```
Comment (jasonburkert-ccpo) [inline diagrams.md:364]: Thanks for maintaining the diagram!

Comment (jasonburkert-ccpo) [reply diagrams.md:364]: Also need to remove relationship near the bottom

`ACQUISITION-PACKAGE ||--|| GFE-OVERVIEW : "TODO delete table"`

**Pull request closed without merging**